### PR TITLE
system-variables: set default tidb_auto_analyze_partition_batch_size as 128

### PR DESCRIPTION
**Pull request by hawkingrei**
…as 128

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tidb/issues/48674

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by qiancai — APPROVED** [review]

**Comment by qiancai** [comment]
@hawkingrei, please involve a tech reviewer for this PR. Thanks.

**Review by winoros — APPROVED** [review]

**Review by winoros — APPROVED** [review]

File: system-variables.md [hunk 1]
```diff
@@ -1173,7 +1173,7 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
 - Scope: GLOBAL
 - Persists to cluster: Yes
 - Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): No
-- Default value: `1`
+- Default value: Before v7.6.0, the default value is `1`. Starting from v7.6.0, the default value is `128`.
```
Comment (qiancai) [inline system-variables.md:1176]:
```suggestion
- Default value: Starting from v7.6.0, the default value is `128`. Before v7.6.0, the default value is `1`. 
```

Comment (qiancai) [inline system-variables.md:1176]:
```suggestion
- Default value: `128`. Before v7.6.0, the default value is `1`.
```

Comment (qiancai) [reply system-variables.md:1176]: 参考了 https://docs.pingcap.com/tidb/dev/system-variables#tidb_analyze_partition_concurrency 的写法
image

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**